src/server/scripts/EasternKingdoms/Scholomance/boss_darkmaster_gandling.cpp

```diff
@@ -302,7 +302,7 @@ class boss_darkmaster_gandling : public CreatureScript
                 SetGate(room, CLOSED);
                 SpawnMobsInRoom(room);
                 DoCast(target, GandlingPortalSpells[room], true); // needs triggered somehow.
-                if (target->GetGUID() == me->GetVictim()->GetGUID())
+                if (target->GetGUID() && target->GetGUID() == me->GetVictim()->GetGUID())
```

Comment:
```suggestion
                if (target && me->GetVictim() && target->GetGUID() == me->GetVictim()->GetGUID())
```

Reply:
This won't do it either because we already check vicitm in the first if statement, L#299
```cpp
            if (spellinfo && spellinfo->Id == SPELL_SHADOW_PORTAL && target && me->GetVictim())